### ISSUE TITLE
Style up Register / Login page

### ISSUE THREAD

**Issue by nikolap994** [Evil-Bees/Booker#19]
Register and Login page are on http://localhost:3000/
We would like to have sliding flow to show register or login form on same page. 
We are using https://mui.com/material-ui/getting-started/overview/ for so there is no need for custom styles.

**Comment by meshiv5** [comment]
Hey , Can i work on this issue

**Comment by nikolap994** [comment]
Sure @meshiv5 :)

**Comment by meshiv5** [comment]
is there any timeline before i have to finish cause i will take some days if not an issue

**Comment by nikolap994** [comment]
Feel free to take as much as you want. If there is anything we can do to help you out, feel free to reach out to us :)

**Comment by meshiv5** [comment]
i would love to get any design reference and to know if u want a responsive design and also if i can use tailwind . i can go with material ui also if thats an issue

**Comment by nikolap994** [comment]
For this project we decided with material Ui. Regarding the design it should be responsive. 
The idea for design is to have 2 forms and on click to switch between them. Main one on page load we would like to show login form and then below it button which will on click "slide" and show Register form and hide Login form.

Does this make sense to you ?

**Comment by meshiv5** [comment]
yes ofcourse i got it . i will update u when i will start working and with progress
